frontend/src/errors.ts

```diff
@@ -4,6 +4,17 @@
 
 import { QueryParamData } from "src/services/search/searchfetcher/SearchFetcher";
 
+export const parseErrorStatus = (error: ApiRequestError): number => {
```

Comment:
note that I don't like that we have to do this, and wonder if there's a better way to pass our errors around that doesn't involve serializing and deserializing JSON